sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -25,33 +35,121 @@ function Build-Project([string]$ProjectName, [string]$SolutionRelativePath) {
         /fl /flp:"logFile=output\$ProjectName.log;verbosity=d"
 }
 
-function Initialize-ActualFolder {
+function Initialize-ActualFolder() {
+    $timer = [system.diagnostics.stopwatch]::StartNew()
+
     Write-Host "Initializing the actual issues Git repo with the expected ones"
     if (Test-Path .\actual) {
         Remove-Item -Recurse -Force actual
     }
+    # this copies no files if ruleId is not set, and all but ending with ruleId if set
 
-    Copy-Item .\expected -Destination .\actual -Recurse
-    Invoke-InLocation "actual" {
-        Exec { git init }
-        Exec { git config user.email "regression-test@example.com" }
-        Exec { git config user.name "regression-test" }
-        Exec { git add -A . }
-        Exec { git commit -m "Initial commit with expected files" | out-null }
-        Get-ChildItem * -Include *.json -Recurse | Remove-Item -Force
-    }
+    CopyFolderRecursively -from .\expected -to .\actual -exclude "*${ruleId}.json"
```

Comment:
[minor] We try to respect the pascal case for arguments: `-From`, `-To` and `-Exclude`.